### DISCUSSION TITLE
Maximum number of clusters per kafka-ui instance

### DISCUSSION THREAD

**Discussion by wanis-fahmy** [provectus/kafka-ui#3058] [Q&A] [answered]
Hey there,

So we're currently planning to provide kafka-ui centrally, deployed on EKS for our teams and initially we are creating a kafka-ui instance per team till the [RBAC](https://github.com/provectus/kafka-ui/issues/753) feature is rolled out. 

We currently have around 50 kafka clusters, assuming RBAC is in place already, is there a maximum amount of clusters to be onboarded on kafka-ui or any limitations one should take into consideration?

Many thanks and keep up the awesome work!

Cheers

**Comment by Haarolean** [comment, marked as answer]
Hey, thanks for reaching out!

There are no limitations and I know a few user setups with a lot of clusters.

If you encounter any problems, just let me know on github or in discord and we'll figure something out :-)
